go-controller/pkg/ovn/egressgw.go

```diff
@@ -438,3 +464,38 @@ func (oc *Controller) delHybridRoutePolicyForPod(podIP net.IP, node string) erro
 	}
 	return nil
 }
+
+func cleanUpBFDEntry(gatewayIP, gatewayRouter string) {
+	portName := types.GWRouterToExtSwitchPrefix + gatewayRouter
+
+	output, stderr, err := util.RunOVNNbctl(
```

Comment:
I thought about that but I wasn't sure it was worth the effort, especially for this case https://github.com/fedepaol/ovn-kubernetes/blob/9ebba002f76bc0df7246dd6ae29b329b96c29509/go-controller/pkg/ovn/egressgw.go#L223 where we don't have the gwInfo object at hand.

This is temporary, when this http://patchwork.ozlabs.org/project/ovn/patch/3c39dc96a36a3445cfa8485a67de79f9f3d5651b.1614602770.git.lorenzo.bianconi@redhat.com/ gets in, we won't need it anymore.

I'll add the doctext, but I'd avoid the cache thing because it adds extra (un-needed, I think) complexity